client/client.go

```diff
@@ -191,22 +191,34 @@ func loadServerTrust(conf Config) (*x509.CertPool, error) {
 		return nil, nil
 	}
 
-	certs, err := x509.SystemCertPool()
+	syscerts, err := x509.SystemCertPool()
+	// fmt.Println(syscerts)
 	if err != nil {
 		return nil, err
 	}
 
 	// Read certificate file.
-	cacert, err := ioutil.ReadFile(conf.ServerCert)
+	servcert, err := ioutil.ReadFile(conf.ServerCert)
+	// fmt.Println(servcert)
 	if err != nil {
 		return nil, err
 	}
-	certs.AppendCertsFromPEM(cacert)
 
-	if len(certs.Subjects()) == 0 {
+	if servcert == nil && syscerts == nil {
```

Comment:
I'm guessing your intention is to handle a case when server certificat exists, but is empty, so `if len(servcert) == 0 { ...`  would be more appropriate.

You create a new cert pool a couple of lines down, so I don't see why checking `syscerts == nil` is needed here.